tests/tt_metal/tt_metal/perf_microbenchmark/dispatch/test_dispatcher.cpp

```diff
@@ -75,12 +90,20 @@ void init(int argc, char **argv) {
         log_info(LogTest, "    -b: dispatcher buffer size in blocks (default {})", DEFAULT_DISPATCH_BUFFER_SIZE_BLOCKS);
         log_info(LogTest, "  -pbs: prefetcher buffer size pages (default {})", DEFAULT_PREFETCHER_BUFFER_SIZE_PAGES);
         log_info(LogTest, " -ppbs: prefetcher page batch size (process pages in batches of N to reduce overhead) (default {})", DEFAULT_PREFETCHER_PAGE_BATCH_SIZE);
-        log_info(LogTest, "  -max: max transfer size bytes (default {})", MAX_XFER_SIZE_16B << 4);
-        log_info(LogTest, "  -min: min transfer size bytes (default {})", MIN_XFER_SIZE_16B << 4);
         log_info(LogTest, "    -f: prefetcher fire once, use to measure dispatcher perf w/ prefetcher out of the way (default disabled)");
-        log_info(LogTest, "    -d: wrap all commands in debug commands (default disabled)");
+        log_info(LogTest, "    -d: wrap all commands in debug commands and clear DRAM (default disabled)");
```

Comment:
not sure what "and clear DRAM" means

Reply:
oh, reading other code I see.  maybe "and clear DRAM to known state"

Reply:
thx, updated message.